### PR TITLE
Apikey better error handling

### PR DESCRIPTION
**Pull request by bencoman**

### CODE REVIEW

File: dev/src/ExercismTools/ExUnauthorized.class.st [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	],
+	#category : #ExercismTools
+}
+
+{ #category : #'instance-creation' }
+ExUnauthorized class >> badToken: tokenStringOrNil [ 
+	^ self new badToken: tokenStringOrNil
+]
+
+{ #category : #'instance creation' }
```
Comment (samWson) [inline dev/src/ExercismTools/ExUnauthorized.class.st]: Minor issue, you've accidentally made two different protocols here: `instance-creation` vs. `instance creation`. Note the `-` between the words.

File: dev/src/ExercismTools/ExUnauthorized.class.st [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ExUnauthorized class >> token: tokenStringOrNil [
+	^self new token: tokenStringOrNil
```
[the remark this answers is not in the document]
Comment (bencoman) [reply dev/src/ExercismTools/ExUnauthorized.class.st]: Good spotting.  Looks like its leftover kruft from renaming   #token:  to   #badToken:
and needs to be deleted.  Could you do those two cleanup and merge?

[the remark this answers is not in the document]
Comment (samWson) [reply dev/src/ExercismTools/ExUnauthorized.class.st]: Will do. I'll be onto that shortly.

File: dev/src/ExercismTools/ExUnauthorized.class.st (hunk 1 above)
Comment (samWson) [reply dev/src/ExercismTools/ExUnauthorized.class.st]: Using `instance-creation` and deleted the other protocol.

**Comment by bencoman** [comment]
Cool. thx Sam.

**Pull request merged**